docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,120 @@
 
 ## Known Issues for PowerShell on Non-Windows Platforms
 
-The first Alpha release of PowerShell on Linux and macOS is mostly functional but
-does have some significant limitations and usability issues.
+Alpha releases of PowerShell on Linux and macOS are mostly functional but do have some significant limitations and usability issues.
+Beta releases of PowerShell on linux and macOS are more functional and stable than alpha releases, but still may be lacking some set of features, and can contain some bugs.
 In some cases, these issues are simply bugs that haven't been fixed yet.
-In other cases (as with the default aliases for ls, cp, etc.) we are
-looking for feedback from the community regarding the choices we make.
+In other cases (as with the default aliases for ls, cp, etc.), we are looking for feedback from the community regarding the choices we make.
 
-Note: Due to the similarities of many underlying subsystems, PowerShell on Linux
-and macOS tend to share the same level of maturity in both features and bugs.
-Except as noted below, the issues in this section will apply to both operating
-systems.
+Note: Due to the similarities of many underlying subsystems, PowerShell on Linux and macOS tend to share the same level of maturity in both features and bugs.
+Except as noted below, the issues in this section will apply to both operating systems.
 
 ### Case-sensitivity in PowerShell
 
 Historically, PowerShell has been uniformly case-insensitive, with few exceptions.
-On UNIX-like operating systems, the file system is case-sensitive and this is exposed through a number
-of ways, obvious and non-obvious.
+On UNIX-like operating systems, the file system is case-sensitive and this is exposed through a number of ways, obvious and non-obvious.
```

Comment:
"the file system is predominantly case-sensitive and PowerShell adheres to the case-sensitivity of the files system; this is exposed..."